lib/test.js

```diff
@@ -52,11 +52,22 @@ function Test (name_, opts_, cb_) {
     this.pendingCount = 0;
     this._skip = args.opts.skip || false;
     this._timeout = args.opts.timeout;
-    this._objectPrintDepth = args.opts.objectPrintDepth || 5;
     this._plan = undefined;
     this._cb = args.cb;
     this._progeny = [];
     this._ok = true;
+    var depthEnvVar = 'NODE_TAPE_OBJECT_PRINT_DEPTH';
+    if (args.opts.objectPrintDepth) {
+        this._objectPrintDepth = args.opts.objectPrintDepth
+    } else if (process.env[depthEnvVar]) {
+        if (process.env[depthEnvVar].toLowerCase() === 'infinity') {
```

Comment:
we should try to be robust against `delete String.prototype.toLowerCase && delete RegExp.prototype.test` (by caching the prototype method at module level)

Reply:
Sorry, I am not sure I understand what you are proposing.